spec/cost_report_workflow_spec.rb

```diff
@@ -83,16 +88,52 @@
       )
     end
 
-    it "runs cost report when all chunks complete" do
+    it "outputs a cost report" do
+      expect { workflow.run }.to change { Dir.glob(cost_report_glob).count }.by(1)
+    end
+
+    it "outputs a frequency table" do
+      expect { workflow.run }.to change { Dir.glob(freqtable_glob).count }.by(1)
+    end
+
+    # TODO: need to add some htitems to get anything useful here
+    it "outputs some data in the cost report" do
+      old_target_cost = Settings.target_cost
+      Settings.target_cost = 16
+      # The data here is only used for the counts of items, not the actual data!
+      # The items in the solr file are all Michigan, and we have no holdings,
+      # so everybody's costs should be zero but Michigan.
+      11.times { load_test_data(build(:ht_item, rights: "ic", collection_code: "PU")) }
+      5.times { load_test_data(build(:ht_item, rights: "pd", collection_code: "PU")) }
+
+      workflow.run
+
+      costreport = File.read(Dir.glob(cost_report_glob).first)
+
+      expect(costreport).to match(/^Total weight: 8.0/m)
+      expect(costreport).to match(/^Num volumes: 16$/m)
+      expect(costreport).to match(/^Num pd volumes: 5$/m)
+      expect(costreport).to match(/^umich\t3.0\t8.0\t0.0.*/m)
+      expect(costreport).to match(/^upenn\t0.0\t0.0\t0.0.*/m)
+    ensure
+      Settings.target_cost = old_target_cost
+    end
+
+    it "outputs the expected frequency table" do
       workflow.run
 
-      expect(false)
-      #      expect(File.exist?("#{ENV["TEST_TMP"]/costreports/:w
-      #
-      #      cr = JSON.parse(File.read("#{ENV["TEST_TMP"]/
+      freqtable = Dir.glob(freqtable_glob).first
+      ft = FrequencyTable.new(data: File.read(freqtable))
+
+      # {"umich"=>{"mpm"=>{"1"=>8}, "spm"=>{"1"=>3}}}
+      expect(ft.frequencies(organization: :umich, format: :mpm))
+        .to contain_exactly(Frequency.new(bucket: 1, frequency: 8))
+
+      expect(ft.frequencies(organization: :umich, format: :spm))
+        .to contain_exactly(Frequency.new(bucket: 1, frequency: 3))
     end
 
-    it "cleans up when cost report completes" do
+    xit "cleans up when cost report completes" do
```

Comment:
If this is a consequence of `CostReportWorkflow::Callback.on_success` (currently) not deleting the intermediate files, I'd like a comment in addition to the test being `xit`ted, lest we forget why.

Reply:
Done.